src/app/meshes/index.ts

```diff
@@ -20,6 +52,9 @@ export const services = (app: Record<string, Token>): ServiceDefinition[] => {
         app.sources,
       ],
     }],
+    ...servicesModule(app),
+    ...dataplanes(app),
+    ...gateways(app),
+    ...policies(app),
```

Comment:
The mesh "micro-frontend"/plugin is the weirdest one in that it has sub dependencies inside it. I wanted to do this so you can just add "mesh" on its own and you get all the sub dependencies automatically.